src/core/pyproxy.js

```diff
@@ -95,7 +96,7 @@ Module.disable_pyproxy_allocation_tracing();
  * still make "prototype in proxy" be true though.
  * @private
  */
-Module.pyproxy_new = function (ptrobj) {
+Module.pyproxy_new = function (ptrobj, cache) {
```

Comment:
Could you please mention cache in the above docstring.

Reply:
Also the existing docstring has a lot of unimportant information about cleaning up callable `PyProxy` objects which is duplicated in comments in the function body, so I'll edit that down.